Configuration.mk

```diff
@@ -40,6 +65,12 @@ endif
 ELF2TAB_ARGS += -n $(PACKAGE_NAME)
 ELF2TAB_ARGS += --stack $(STACK_SIZE) --app-heap $(APP_HEAP_SIZE) --kernel-heap $(KERNEL_HEAP_SIZE)
 
+# Setup the correct toolchain for each architecture.
+override TOOLCHAIN_cortex-m0 = arm-none-eabi
+override TOOLCHAIN_cortex-m3 = arm-none-eabi
+override TOOLCHAIN_cortex-m4 = arm-none-eabi
+override TOOLCHAIN_rv32imac = riscv64-unknown-elf
```

Comment:
It looks like the Debian toolchain insteads 32-bit support in the 64-bit toolchain, but not all setups would do that. It might be worth documenting that this need to be edited if that's the case.

Reply:
We can't have users edit makefiles, we either need to have instructions on how to install the "correct" binaries, or have the makefiles support both.

Reply:
Ok, it probably is easiest to default to `riscv32-unknown-elf` and fallback to `riscv64-unknown-elf`. That should cover all bases.